packages/CollabCodeEdit-Core.package/CCEMethodRecategorizedEdit.class/properties.json

```diff
@@ -0,0 +1,14 @@
+{
+	"category" : "CollabCodeEdit-Core",
+	"classinstvars" : [
+		 ],
+	"classvars" : [
+		 ],
+	"commentStamp" : "AD 6/24/2019 11:44",
+	"instvars" : [
+		 ],
+	"name" : "CCEMethodRecategorizedEdit",
+	"pools" : [
+		 ],
+	"super" : "AnObsoleteCCEEdit",
```

Comment:
This should be `CCEEdit`. Please check, that the `CCEEdit` is not deleted in your branch.